### PR TITLE
Adding release info for my future self.

### PR DESCRIPTION
**Pull request by skeet70**

### CODE REVIEW

**Comment by leifwickland** [comment]
I love it!

**Review by leifwickland — APPROVED** [review]

File: src/main/tut/README.md [hunk 1]
```diff
@@ -226,3 +226,10 @@ Run `sbt test` to build the project and exercise the unit test suite.
 Rubiz is versioned with [semver](http://semver.org/) and released to [sonatype](https://oss.sonatype.org/).
 Open an issue on Github if you feel there should be a release published that hasn't been. You can run `sbt tut`
 to regenerate documentation locally if you modify the docs in the `/main/tut` directory.
+
+For those with permission to release:
+* Install `gpg` and [generate a key](https://www.gnupg.org/gph/en/manual.html#AEN26). Upload that key to a [public key server](http://pgp.mit.edu/).
+    * Mac users can `brew install gpg pinentry-mac` to get the tools needed.
+* Create a Sonatype [credentials file](http://www.scala-sbt.org/1.0/docs/Using-Sonatype.html#Fourth+-+Adding+credentials).
+* Run `sbt release`
+* `git push` to add the new tags and release commits to master.
```
Comment (coltfred) [inline src/main/tut/README.md:235]: This isn't all you need to do.
 
`git push upstream master` (assuming you have it named upstream)

Comment (skeet70) [reply src/main/tut/README.md:235]: Hmm... I assumed the release was being run from the main repo, so I didn't use upstream. Is that incorrect?

Comment (coltfred) [reply src/main/tut/README.md:235]: Your assumption is that all users that can publish don't have their own forks, which isn't the way people should do development on OSS projects (IMO). Maybe you shouldn't give a command and instead say they should push the tags and version bumps to the `rubicon-project/rubiz`

Comment (skeet70) [reply src/main/tut/README.md:235]: My assumption was that we only want our releases to be off the `master` of `rubicon-project/rubiz`. Not that other work doesn't happen in forks. I'll change to the more flexible wording though.

**Pull request merged**